src/metrics_server/README.md

```diff
@@ -35,6 +35,7 @@ The metrics server supports two URL paths:
         timestampUtcMs: number,
         dataLimit: {
             enabled: boolean
+            perKeyLimitCount: number
```

Comment:
I'm not sure the difficulties you're worried about -- could you elaborate?  I figured this way would make more sense since we've called it "dataLimit", not "serverDataLimit"``.

Reply:
Nothing concrete, but in general it's a good idea to not couple things that can be decoupled. The two limits are enabled independently.
I don't feel strongly though.